### PR TITLE
Removed the high and low gear ticks per inch property

### PR DESCRIPTION
**Pull request by imathno**
Added one ticks per inch property

### CODE REVIEW

File: Competition2017/src/competition/subsystems/drive/DriveSubsystem.java [hunk 1]
```diff
@@ -59,25 +54,21 @@ public DriveSubsystem(WPIFactory factory, XPropertyManager propManager, ShiftSub
         leftDriveEncoderTicksProp = propManager.createEphemeralProperty("Left drive encoder ticks", 0);
         rightDriveEncoderTicksProp = propManager.createEphemeralProperty("Right drive encoder ticks", 0);
                 
-        startPositionTicks = propManager.createEphemeralProperty("Start position ticks", 0);
```
Comment (WasabiFan) [inline Competition2017/src/competition/subsystems/drive/DriveSubsystem.java:62]: What was this property?

**Review by WasabiFan — APPROVED** [review]

File: Competition2017/src/competition/subsystems/drive/DriveSubsystem.java (hunk 1 above)
Comment (imathno) [reply Competition2017/src/competition/subsystems/drive/DriveSubsystem.java:62]: It was a property used along time ago for isFinished but isn't used anymore.

File: Competition2017/src/competition/subsystems/drive/DriveSubsystem.java [hunk 2]
```diff
@@ -4,7 +4,6 @@
 import com.google.inject.Singleton;
 
 import competition.subsystems.shift.ShiftSubsystem;
```
Comment (aschokking) [inline Competition2017/src/competition/subsystems/drive/DriveSubsystem.java:6]: unused import now?

**Comment by aschokking** [comment]
Were there no tests on the 2 properties that need updating now?

File: Competition2017/src/competition/subsystems/drive/DriveSubsystem.java (hunk 2 above)
Comment (imathno) [reply Competition2017/src/competition/subsystems/drive/DriveSubsystem.java:6]: Yeah it was used for the enum

**Pull request merged**